digital_land/commands.py

```diff
@@ -403,6 +403,33 @@ def collection_add_source(entry, collection, endpoint_url, collection_dir):
     add_source_endpoint(entry, directory=collection_dir)
 
 
+def add_endpoints_and_lookups(ctx):
+    """
+    perform required tasks to add new endpoints to a collection
+    :param ctx:
+    :return:
+    """
+    from digital_land.utils.add_endpoints_utils import (
+        task_preprocess,
+        task_create_source_and_endpoint_entries,
+        task_collect_resources,
+        task_populate_resource_and_log_csvs,
+        task_generate_lookup_entries,
+        task_update_lookup_csv,
+        task_postprocess,
+    )
+
```

Comment:
This is very hard to read as each function is only useful for this purpose. Creating and using classes will be readable as people can read what your doing.